test/fcoe_client_write_spec.rb

```diff
@@ -16,33 +16,70 @@
   end
 
   describe "#WriteSysconfigFiles" do
-    context "for odd cases" do
-      before do
-        interfaces = [
-          { "fcoe_vlan" => "not available" },
-          { "fcoe_vlan" => "not configured" },
+    before do
+      allow(subject).to receive(:GetNetworkCards).and_return(interfaces)
+    end
+
+
+    context "when no VLAN was created for any of the interfaces" do
+      let(:interfaces) do
+        [
+          { "dev_name" => "eth0", "fcoe_vlan" => "not available" },
+          { "dev_name" => "eth1", "fcoe_vlan" => "not configured" },
+          { "dev_name" => "eth2", "fcoe_vlan" => "" },
         ]
-        expect(subject).to receive(:GetNetworkCards).and_return(interfaces)
       end
 
       it "smokes not" do
         expect { subject.WriteSysconfigFiles }.to_not raise_error
       end
+
+      it "writes nothing into /etc/sysconfig/network" do
+        expect(Yast::SCR).to_not receive(:Write)
+          .with(path_matching(/^\.network\..*/), anything)
+
+        subject.WriteSysconfigFiles
+      end
     end
 
-    context "for a small FCoE setup" do
-      before do
-        interfaces = [
-          { "fcoe_vlan" => "eth1.500" }
+    context "if an FCoE VLAN is created for some interface" do
+      let(:interfaces) do
+        [
+          { "dev_name" => "eth0", "fcoe_vlan" => "not available" },
+          { "dev_name" => "eth1", "fcoe_vlan" => "eth1.500-fcoe" },
```

Comment:
np: Maybe we should also update the documentation with the suffix -fcoe option

https://github.com/yast/yast-fcoe-client/blob/master/src/modules/FcoeClient.rb#L49

Reply:
Done